### PR TITLE
add bebop parser

### PR DESCRIPTION
**Pull request by ngocthanh1389**

### CODE REVIEW

File: internal/worker/worker.go [hunk 1]
```diff
@@ -128,7 +128,7 @@ func (w *Worker) processMessages(m []evmlistenerclient.Message) error {
 
 func (w *Worker) retryParseLog() error {
 	insertOrders := []storage.TradeLog{}
-	logs, err := w.s.GetErrorLogsSince(time.Now().Add(-time.Hour * 24).Unix())
+	logs, err := w.s.GetErrorLogsSince(time.Now().Add(-time.Minute * 5).Unix())
```
Comment (Haiss2) [inline internal/worker/worker.go:131]: RetryTime is 2 hours, should we change value to be ≥ 2h?

File: pkg/parser/bebop/parser.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+		}
+		var rfqOrder SingleOrder
+		bytes, err := json.Marshal(param.Value)
+		if err != nil {
+			return order, err
+		}
+
+		if err := json.Unmarshal(bytes, &rfqOrder); err != nil {
+			return order, err
+		}
```
Comment (Haiss2) [inline pkg/parser/bebop/parser.go:157]: Did you try:
```go
rfqOrder, ok := param.Value.(SingleOrder)

Comment (Haiss2) [reply pkg/parser/bebop/parser.go:157]: I tried, it doesn't works. Maybe your code is the best solution

File: internal/worker/worker.go (hunk 1 above)
Comment (ngocthanh1389) [reply internal/worker/worker.go:131]: Because it spam node now, almost error parsed log can't rety

**Review by Haiss2 — APPROVED** [review]
LGTM

**Pull request merged**